### PR TITLE
feat: referendum delegated vote indicator

### LINKED ISSUE #1984: OpenGov. Referendums. Show if the account voted via delegate
OpenGov. Referendums. Show if the account voted via delegate

If the user account has delegated votes and the delegate has voted for the referenda then the Voted badge with Delegate info has to be shown on the list of referendas.

The information about delegation should be read from SubQuery.

Figma [mockup](https://www.figma.com/design/4vjREBpvZK7M3m1GaoVptq/OpenGov-delegations?node-id=1-5558&t=0wTTRykyxOWeqxyE-4)

NovaWallet's query to SubQuery:
```js
{
    castingVotings(filter: { voter: {equalTo: "13mAjFVjFDpfa42k2dLdSnUyrSzK8vAySsoudnxX2EKVtfaq"}}) {
        nodes {
            referendumId
            standardVote
            splitVote
            splitAbstainVote
        }
    }

    delegatorVotings(filter: {delegator: {equalTo: "13mAjFVjFDpfa42k2dLdSnUyrSzK8vAySsoudnxX2EKVtfaq"}}) {
        nodes {
            vote
            parent {
                referendumId
                voter
                standardVote
            }
        }
    }
}
```

### PR DESCRIPTION
**Pull request by johnthecat**
Снимок экрана 2024-08-14 в 18 56 13

### CODE REVIEW

File: src/renderer/entities/network/model/network-model.ts [hunk 1]
```diff
@@ -240,7 +253,14 @@ sample({
...
 
-    return { chainId, nodes, metadata, providerType };
+    return {
+      chainId,
+      nodes,
+      metadata,
+      providerType,
+      // set to true in case of soe network issues
+      DEBUG_NETWORKS: false,
+    };
```
Comment (sokolova-an) [inline src/renderer/entities/network/model/network-model.ts]: better to announce these changes on standups so everyone knew where to find it 
should there be 'some network issues'?

File: src/renderer/features/governance/aggregates/delegatedVotes.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  networkSelectorModel.$governanceChain,
+  (votes, chain) => {
+    if (!chain) return {};
+
+    return votes[chain.chainId] ?? {};
+  },
+);
+
+sample({
+  clock: [referendumModel.events.requestDone, networkSelectorModel.$governanceChainApi, walletModel.$activeWallet],
```
Comment (sokolova-an) [inline src/renderer/features/governance/aggregates/delegatedVotes.ts:23]: is it possible for this clock to be triggered if we have `networkSelectorModel.$governanceChainApi`, but already left the gov page and switching between wallets?

File: src/renderer/entities/network/model/network-model.ts (hunk 1 above)
Comment (johnthecat) [reply src/renderer/entities/network/model/network-model.ts]: You're right, I'll announce it

File: src/renderer/features/governance/aggregates/delegatedVotes.ts (hunk 2 above)
Comment (johnthecat) [reply src/renderer/features/governance/aggregates/delegatedVotes.ts:23]: delegated votes are separated by chains, so it's same behavior as anywhere in gov(

**Review by sokolova-an — APPROVED** [review]

**Pull request merged**